books/free-programming-books-hi.md

```diff
@@ -1,10 +1,17 @@
 ### Index
 
 * [C++](#cpp)
+* [Java](#java)
+* [Python](#python)
 
 
 ### <a id="cpp"></a>C++
 
 * [C++ Brief Notes \| Hindi](https://ehindistudy.com/2020/12/01/cpp-notes-in-hindi/) - Yugal Joshi
 * [C++ in Hindi \| Hindi](https://www.bccfalna.com/IOC-AllEBooks/CPPinHindi.pdf) - Kuldeep Chand (PDF)
 * [C++ Introduction Book \| Hindi](https://ncsmindia.com/wp-content/uploads/2012/04/c++-hindi.pdf) - NCMS India (PDF)
+
```

Comment:
The changes in this file are not required

Reply:
I have just removed the links that were related to Google Drive.

Reply:
also revert the changes you made in the books-hi index. thanks

Reply:
I have done it.